### PR TITLE
Customized TF estimator: pair matching with 1st order correction

### PR DESCRIPTION
**Pull request by wcchu**

### CODE REVIEW

File: PairLinCorr/pairlincorr.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        'item2', vocabulary_list=list_items))
+value2_col = tf.feature_column.numeric_column('value2')
+
+
+# build input function for training
+input_fn_train = tf.estimator.inputs.pandas_input_fn(
+    x=x_train,
+    y=y_train,
+    batch_size=100,
+    num_epochs=3,
```
Comment (wcchu) [inline PairLinCorr/pairlincorr.py]: Higher `num_epochs` will reduce loss but we keep it to 3 here for convenience of development.

**Comment by wcchu** [comment]
[Customized TF estimator: pair matching with 1st order correction](https://trello.com/c/uWtrfMr6/21-customized-tf-estimator-pair-matching-with-1st-order-correction)

File: PairLinCorr/pairlincorr.py [hunk 2]
```diff
@@ -157,42 +140,23 @@ def model_fn(features, labels, mode, params):
...
+checkpoint = tf.train.get_checkpoint_state(checkpoint_dir=MODEL_DIR)
+with tf.Session() as sess:
+    saver = tf.train.import_meta_graph(
+        checkpoint.model_checkpoint_path + '.meta')
+    saver.restore(sess, checkpoint.model_checkpoint_path)
+    tvs = sess.run(tf.trainable_variables())
+
+
+# construct and export coefficient table
+coeffs = np.reshape(tvs, [2, 704])
```
Comment (LisaBM) [inline PairLinCorr/pairlincorr.py]: Why do you use 704 here? Is that the number of items?

Comment (wcchu) [reply PairLinCorr/pairlincorr.py]: Yes that's the number of items. I actually removed the `.py` file in the middle of this pull request and only developed on `.ipynb` later...sorry for saying this just now.

Comment (wcchu) [reply PairLinCorr/pairlincorr.py]: In fact it's removed in the last step, alright I'm not that sorry now.

**Review by LisaBM — APPROVED** [review]
